pkg/controller/kubelet-config/kubelet_config_bootstrap_test.go

```diff
@@ -199,8 +201,7 @@ func TestAddKubeletCfgAfterBootstrapKubeletCfg(t *testing.T) {
 	for _, platform := range []configv1.PlatformType{configv1.AWSPlatformType, configv1.NonePlatformType, "unrecognized"} {
 		t.Run(string(platform), func(t *testing.T) {
 			f := newFixture(t)
-
-			fgAccess := createNewDefaultFeatureGateAccess()
+			fgAccess := featuregates.NewHardcodedFeatureGateAccess([]osev1.FeatureGateName{"AlibabaPlatform"}, nil)
```

Comment:
I see `AlibabaPlatform` used in all these tests is this arbitrary or part of the test?

Reply:
yes this is arbitrary. when I was initially testing, I used more custom hardocded feature gates, and they passed. I changed to `Alibaba` for simplicity and succinctness. If anyone wants a specific custom feature gate, LMK

Reply:
One potential concern I would have is that featuregates come and go, so instead of a hard-coded arbitrary list/item, it may be easier to maintain if we can somehow grab the current set of available FGs so we don't have to change this (unless we decide to massively change the way FGs are used, but that won't happen again right ;)